app/src/main/java/org/stepic/droid/ui/fragments/ProfileFragment.kt

```diff
@@ -283,7 +286,19 @@ class ProfileFragment : FragmentBase(),
                 shortBioFirstHeader.setText(R.string.short_bio_and_info)
                 shortBioFirstText.text = shortBio
                 shortBioSecondHeader.setText(R.string.user_info)
-                shortBioSecondText.text = information
+
+                val (description, isNeedWebView, _) = textResolver.resolveStepText(information)
```

Comment:
information can be in `shortBioFirstText`, when it is not blank, but `shortBio` is blank